src/test/java/se/sundsvall/supportmanagement/service/mapper/ErrandMapperTest.java

```diff
@@ -511,6 +513,22 @@ void testUpdateEntityWithBlank() {
 		assertThat(entity.getContactReasonDescription()).isNull();
 	}
 
+	@Test
+	void testingSTuff(){
+
+		var errand = updateEntity(new ErrandEntity().withSuspendedTo(now()).withSuspendedTo(now().plusDays(5)), Errand.create().withSuspension(Suspension.create()));
+
+var dummy = Errand.create().withSuspension(Suspension.create());
+		Gson gson = new GsonBuilder()
+			.registerTypeAdapter(OffsetDateTime.class, OffsetDateTimeSerializer.create())
+			.addSerializationExclusionStrategy(CircularReferenceExclusionStrategy.create())
+			.addSerializationExclusionStrategy(AttachmentExclusionStrategy.create())
+			.create();
+	var json =	gson.toJson(dummy);
+		System.out.printf("json: %s", json);
+		assertThat(errand.getSuspendedFrom()).isNull();
+	}
```

Comment:
Not quite sure if this test is meant to be left in the code or if it was used during development. If it is supposed to be here, it could use a better name and removal of ```System.out...```